addon/components/ember-sparkles/pie-chart.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import layout from '../../templates/components/ember-sparkles/pie-chart';
+
+export default Ember.Component.extend({
+  layout,
+  radius: Math.min(this.width, this.height) / 2,
```

Comment:
this will most likely not live in any JS file once we get there, Albert is working on this branch to build a pie chart, he has just begun.